crossbar/controller/process.py

```diff
@@ -528,7 +528,10 @@ def on_ready_success(id):
                 'who': worker.who
             }
 
-            self.publish(started_topic, started_info, options=PublishOptions(exclude=[details.caller]))
+            options = None
+            if details.caller is not None:
+                options = PublishOptions(exclude=[details.caller])
+            self.publish(started_topic, started_info, options=options)
 
```

Comment:
I'll rebase and re-submit, as code like this can now be simplified back to what it was thanks to recent AutobahnPython merge.